ui/resizable.js

```diff
@@ -1174,6 +1174,13 @@ $.ui.plugin.add("resizable", "grid", {
 
 });
 
+// DEPRECATED
+if ( $.uiBackCompat !== false ) {
+
+	// Ghost option
+	// TODO add backcompat once i know how $.ui.plugin works...
```

Comment:
Yup sorry i meant to include this along with the other 2 options that need to move to classes `hoverClass` and `activeClass`